docs/changehistory/Roadmap.md

```diff
@@ -7,7 +7,7 @@ This roadmap provides the *big picture* view of the development path for the iTw
 We will use the following themes (that may be adjusted from time-to-time according to your input) to drive the evolution of iTwin.js:
 
 - Be *the open platform* for Infrastructure Digital Twins for federating BIM Models, GIS data, reality data, IT and OT into a single pane of glass for a seamless user experience.
```

Comment:
@ColinKerr is "single pane of glass" your terminology? I don't know what it means - do other readers?

Reply:
No.  'Glass' gets used in some of our UX descriptions but I agree it is not appropriate here and should be changed.

I would suggest: ... IT and OT together for a seamless user experience.